pom.xml

```diff
@@ -25,7 +25,7 @@
     <version>30</version>
   </parent>
   <artifactId>gwt-maven-plugin</artifactId>
-  <version>2.7.0-SNAPSHOT</version>
+  <version>2.7.0-coverage-SNAPSHOT</version>
```

Comment:
Sorry, I did not see your comment earlier!
I changed the artifact id on my branch to avoid conflict while testing.